docs/sql-data-sources-avro.md

```diff
@@ -353,6 +353,13 @@ Data source options of Avro can be set via:
     <td>read</td>
     <td>4.0.0</td>
   </tr>
+  <tr>
```

Comment:
Thanks for adding the doc for the new option! Apart from this, should we add a block on recursion support similar to what we do for [protobuf](https://github.com/apache/spark/blob/34e65a8e72513d0445b5ff9b251e3388625ad1ec/docs/sql-data-sources-protobuf.md?plain=1#L402-L434)?

Reply:
Good idea.